presto-orc/src/main/java/com/facebook/presto/orc/MissingSubscriptException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+public class MissingSubscriptException
+        extends RuntimeException
+{
+    public MissingSubscriptException()
+    {
+    }
+
+    @Override
+    public String getMessage()
+    {
+        return "Array subscript out of bounds or map key not present";
```

Comment:
Prior comment:

---
It would be helpful to include the subscript or key that's missing.